README.md

```diff
@@ -672,6 +672,14 @@ Circus Train can be extended in various ways. This is an advanced feature that t
 * `LocationManagers` look after the source and replica locations and are an ideal point to implement both snapshot isolation and retired data clean-up.
 * `CompositeCopierFactory` allows the provision of multiple copiers for the same table in order to add functionality to the copy process, e.g. introduce compaction, copy to multiple destinations, etc.
 
+### External Extensions
+
+#### Circus Train DataSqueeze Copier
+The [DataSqueeze](https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze) extension provides new Copiers for Circus Train which compact the data being replicated in various ways.
+
+#### circus-train-bigquery
```

Comment:
I think this can be "Circus Train BigQuery" since the one above isn't "circus-train-datasqueeze" :)

Reply:
Agreed, I thought the same thing earlier, I'll fix that quickly now.